source/agora/test/TimeDrift.d

```diff
@@ -37,14 +36,6 @@ unittest
     // Make sure nodes have revealed their preimage for height 1
     network.waitForPreimages(GenesisBlock.header.enrollments, Height(1));
 
-    // sanity check for the generated quorum config
-    nodes.enumerate.each!((idx, node) =>
-        retryFor(node.getQuorumConfig().threshold == conf.consensus.max_quorum_nodes &&
-                node.getQuorumConfig().nodes.length == conf.consensus.max_quorum_nodes,
-            5.seconds,
-            format("Node #%s has invalid quorum config for test: %s",
-                idx, node.getQuorumConfig())));
-
```

Comment:
Instead of removing this, can't we use the `quorumSize` in the code?  But if you think removing the code would be better, I have no objection.

Reply:
I think now that the quorum size is calculated then this check is a bit redundant.